### PR TITLE
Breakout HVAC ODU Sensors

### PR DESCRIPTION
**Pull request by Daniel-dev22**
Not all HVAC systems will have communicating outdoor units.

Another pr to follow adding IDs to sensors and additional sensors for both base HVAC and odu.

### CODE REVIEW

File: example-esp8266.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ packages:
     # Change this file to the correct one for your device
     file: econet_heatpump_water_heater.yaml
     # file: econet_hvac.yaml
+    # Use this if you have a comminicating outdoor unit (ODU)
+    # files: [econet_hvac.yaml, econet_hvac_odu.yaml]
```
Comment (andrejpavlovic) [inline example-esp8266.yaml:16]: This affects people who do have ODU through right? They will need to comment out `econet_hvac.yaml` and uncomment `files: [econet_hvac.yaml, econet_hvac_odu.yaml]` in order to get the same results?

Probably not many people, but just something to consider :p

File: econet_hvac_odu.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
```
Comment (tronikos) [inline econet_hvac_odu.yaml:1]: > 1:4 [new-lines] wrong new line character: expected \n
Can you convert this to using UNIX new lines? If you can run pre-commit it should take it care for you. Probably easier to open this file in Notepad++ right click at the bottom right corner where it says "Windows (CR LF)" and change it to "Unix (LF)"

**Comment by tronikos** [comment]
There are many unrelated commits. Once merged they will be part of the commit history. Probably better to start fresh from a clean branch.

**Comment by barndawgie** [comment]
You can use a rebase in the future to help solve that. E.g. `git rebase -i HEAD~7` will let you squash the last 7 commits quite easily.

**Pull request closed without merging**